pages/studentAssessmentHomework/studentAssessmentHomework.sql

```diff
@@ -46,42 +46,39 @@ WITH log AS (
 )
 INSERT INTO group_logs
     (authn_user_id, user_id, group_id, action)
-SELECT $user_id, $user_id, group_id, 'join'
+SELECT $authn_user_id, $user_id, group_id, 'join'
 FROM log;
 
 
 -- BLOCK create_group
-WITH log AS (
+WITH
+create_group AS (
     INSERT INTO groups
         (name, group_config_id, course_instance_id)
-    VALUES
-        (
-            $group_name,
-            (SELECT id FROM group_configs WHERE assessment_id = $assessment_id AND deleted_at IS NULL),
-            (SELECT course_instance_id FROM group_configs WHERE assessment_id = $assessment_id AND deleted_at IS NULL)
-        )
+    (
+        SELECT 
+            $group_name, id, course_instance_id
+        FROM 
+            group_configs
```

Comment:
Would you mind making this `group_configs AS gc` and putting an explicit `gc.` on the front of all column references? That's much easier for me to check, and I think it's generally safer (less likely to accidentally reference the wrong `id` column, for example).